pyproject.toml

```diff
@@ -1,7 +1,7 @@
 [build-system]
 requires = [
     "setuptools>=66.1",
-    "setuptools_scm>=7"
+    "setuptools_scm>=8.1"
```

Comment:
The same change has to be implemented in the noxfile

Reply:
Done.